lib/alex_koin/slack.ex

```diff
@@ -60,21 +79,35 @@ defmodule AlexKoin.SlackRtm do
 
     # validate this coin belongs to the user currently
     if coin.wallet_id == user_wallet(user).id do
-      SlackCommands.transfer_coin(coin, user_wallet(user), user_wallet(to_user), captures["memo"])
+      memo = captures["memo"]
```

Comment:
I'm an idiot. instead of this change, we should change line 74 to a pattern match: 

```elixir
%{"memo" => memo} = Regex.named_captures(regex, text)
```